### PR TITLE
Build and bump version for 1.2.2

### PR DESCRIPTION
**Pull request by macandcheese**
Give this a 👀 over

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ## [1.2.2]
```
Comment (paulcpederson) [inline CHANGELOG.md]: In order to make this work we need to add this link definition at the bottom of the file like:

```
[1.2.1]: https://github.com/esri/calcite-web/compare/v1.2.1...v1.2.2
```

That turns the header into a link to the diff

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
 ### Fixes
--  Updates positioning mixins to unset directional values in RTL
+-  Fixes for switch and radio group
+-  Updates positioning mixin
 -  Updates checkbox and radio alignment in Edge
```
Comment (paulcpederson) [inline CHANGELOG.md]: we could probably remove this extra space too

**Comment by paulcpederson** [comment]
couple little nitpicks with the `CHANGELOG` otherwise looks good.

**Comment by macandcheese** [comment]
That's a Texas sized 10-4, changes added.

**Comment by paulcpederson** [comment]
will merge once travis gives me the ol green light

**Pull request merged**